src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -248,6 +248,10 @@ export class CommandDetectionCapability implements ICommandDetectionCapability {
 		this._isWindowsPty = value;
 	}
 
+	disableCommandStorage(): void {
```

Comment:
Closer naming to the other set functions: `setIsCommandStorageDisabled and `_isCommandStorageDisabled`, the variable also then does not start with verb which makes it look like a method.